cypress-tests/cypress/e2e/PaymentTest/00020-MandatesUsingNTID.cy.js

```diff
@@ -0,0 +1,221 @@
+import * as fixtures from "../../fixtures/imports";
+import State from "../../utils/State";
+import getConnectorDetails, * as utils from "../PaymentUtils/Utils";
+
+let globalState;
+let connector;
+
+describe("Card - Mandates using Network Transaction Id flow test", () => {
+  before("seed global state", () => {
+    cy.task("getGlobalState").then((state) => {
+      globalState = new State(state);
+      connector = globalState.get("connectorId");
+    });
+  });
+
+  afterEach("flush global state", () => {
+    cy.task("setGlobalState", globalState.data);
+  });
+
+  context(
+    "Card - NoThreeDS Create and Confirm Automatic MIT payment flow test",
+    () => {
+      let should_continue = true;
+
+      beforeEach(function () {
+        if (!should_continue || connector !== "cybersource") {
+          this.skip();
+        }
+      });
+
+      it("Confirm No 3DS MIT", () => {
+        cy.mitUsingNTID(
+          fixtures.ntidConfirmBody,
+          7000,
+          true,
+          "automatic",
+          globalState
+        );
+      });
+    }
+  );
+
+  context(
+    "Card - NoThreeDS Create and Confirm Manual MIT payment flow test",
+    () => {
+      let should_continue = true;
+
+      beforeEach(function () {
+        if (!should_continue || connector !== "cybersource") {
+          this.skip();
+        }
+      });
+
+      it("Confirm No 3DS MIT", () => {
+        cy.mitUsingNTID(
+          fixtures.ntidConfirmBody,
+          7000,
+          true,
+          "automatic",
```

Comment:
context says confirm manual and we are passing automatic as capture method ? can you make it as manual and add capture flow